userland/userland_generic.ld

```diff
@@ -109,16 +113,19 @@ SECTIONS {
     .stack :
     {
         . += STACK_SIZE;
+        . = ALIGN(4); /* Make sure we're word-aligned here */
```

Comment:
I don't think we need the stack or heap aligns as these are only hints/minimums. The underlying kernel is permitted to grow these to satisfy platform requirements anyway (these "sections" really only exist so that elf2tbf can parse them out. They probably should be marked as `NOLOAD` though)